docs/notary/setup-Komodo-Notary-Node.md

```diff
@@ -489,8 +495,14 @@ berkeleydb () {
     wget -N 'http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz'
     echo '12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef db-4.8.30.NC.tar.gz' | sha256sum -c
     tar -xzvf db-4.8.30.NC.tar.gz
-    # Uncomment the following line in case of Debian 10
-    # sed -i 's/__atomic_compare_exchange/__atomic_compare_exchange_db/g' db-4.8.30.NC/dbinc/atomic.h
+  if [ -f /etc/debian_version ]; then
```

Comment:
looks like ident is missing there, otherwise looks fine if it was tested on Debian

Reply:
updated